sdk/containerregistry/azure-containers-containerregistry/pom.xml

```diff
@@ -58,6 +58,12 @@
       <artifactId>azure-core-http-netty</artifactId>
       <version>1.13.3</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-test</artifactId>
+      <version>1.16.1</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
```

Comment:
we will need the latest released core version which has the support for asset-sync feature.